Makefile

```diff
@@ -1,28 +1,57 @@
-all: init build up
+NAME = dina-web/collections-data-model
+VERSION = $(TRAVIS_BUILD_ID)
+ME = $(USER)
+HOST = beta.dina-web.net
+TS := $(shell date '+%Y_%m_%d_%H_%M')
+
 
+all: init build up
 
 init:
-	@echo "Pulling code from GitHub"
-	# cd cco-tools && git clone https://github.com/Inkimar/cco_poc.git
-	cd cco-tools && git clone https://github.com/DINA-Web/cco_poc.git
+	mkdir -p mysql-datadir backups
 
 build:
+	@echo "Pulling source code for dependencies from GitHub"
+	cd cco-tools && git clone https://github.com/DINA-Web/cco_poc.git
+	cd cco-tools && curl -L -s -o wait-for-it.sh \
+		https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh && \
+		chmod +x wait-for-it.sh
 	docker-compose build cco
 
 up: 
-	@echo "Sorry for the 30s below, please fix it with a PR ;)"
+	@echo "Starting database and creating schema..."
 	docker-compose up -d db
-	sleep 30 && docker-compose run cco
+	docker-compose run cco
 
 connect:
-	@echo "Connecting to CCO database and display the tables"
-	mysql -u dina -ppassword12 -P 13306 -h 127.0.0.1 cco -e "show tables"
+	@export $(cat .env | xargs) > /dev/null
+	@echo "Connecting to db... pls enter 'show tables' at the prompt."
+	docker-compose run --rm db sh -c \
```

Comment:
Could this command be executed inside the container to avoid having to install the mysql command line tool?